mmv1/third_party/terraform/resources/resource_compute_router_interface.go.erb

```diff
@@ -47,28 +47,40 @@ func resourceComputeRouterInterface() *schema.Resource {
 			},
 			"vpn_tunnel": {
 				Type:             schema.TypeString,
-				ConflictsWith:    []string{"interconnect_attachment"},
 				Optional:         true,
 				ForceNew:         true,
 				DiffSuppressFunc: compareSelfLinkOrResourceName,
-				AtLeastOneOf:     []string{"vpn_tunnel", "interconnect_attachment", "ip_range"},
+				ConflictsWith:    []string{"interconnect_attachment", "subnetwork"},
 				Description:      `The name or resource link to the VPN tunnel this interface will be linked to. Changing this forces a new interface to be created. Only one of vpn_tunnel and interconnect_attachment can be specified.`,
```

Comment:
These doc strings will need to be updated too, KCC draws from them.

Reply:
What doc strings are you referring to?

Reply:
If you're referring to the doc, I think I did (see the doc file modified). Otherwise, I'm missing some other file. In case, could you please point me to it?

Reply:
Ah, gotcha! Done. Sorry I completely missed this